packages/amplify-appsync-simulator/API.md

```diff
@@ -95,7 +95,7 @@ export class AmplifyAppSyncSimulator {
     // (undocumented)
     start(): Promise<void>;
     // (undocumented)
-    stop(): void;
+    stop(): Promise<void>;
```

Comment:
We need to make sure that this is not a breaking change with the data team. I believe we need to reach out and get alignment on this.